packages/hero/src/scss/_component.scss

```diff
@@ -25,8 +25,12 @@
 
   &__text {
     
-    & h1 {
+    & h1,
```

Comment:
Just test, but I don't think you need the H1 here, regardless of selector used the class will override.